tests/appstreamcli_validate.pm

```diff
@@ -0,0 +1,31 @@
+# vi: set shiftwidth=4 tabstop=4 expandtab:
+use base 'installedtest';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    # Start from the desktop
+    check_desktop_clean();
```

Comment:
This test shouldn't need a graphical login, right?

Reply:
All the tests have it and I can’t remember if things work without it, but I’ll try without.

Reply:
OK either way, just wondering – removing it would mean that an unclean desktop would not block this test.

Reply:
Removing it worked.